src/components/structures/CreateRoom.js

```diff
@@ -249,13 +249,15 @@ module.exports = React.createClass({
             var RoomAlias = sdk.getComponent("create_room.RoomAlias");
             var Presets = sdk.getComponent("create_room.Presets");
             var UserSelector = sdk.getComponent("elements.UserSelector");
-            var RoomHeader = sdk.getComponent("rooms.RoomHeader");
+            var SimpleRoomHeader = sdk.getComponent("rooms.SimpleRoomHeader");
 
             var domain = MatrixClientPeg.get().getDomain();
 
             return (
             <div className="mx_CreateRoom">
-                <RoomHeader simpleHeader="Create room" />
+                <SimpleRoomHeader>
+                    Create room
```

Comment:
Hm, I'm quite unconvinced by passing in the title as the contents of the tag.  I'd expect simple room headers to have structure for (optional) avatar and title at the least - at which point i'd expect to be passing them in as properties.  Is there a rationale i'm missing?